### PR TITLE
refactoring to prepare for restructure

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: yk [hunk 1]
```diff
@@ -264,6 +279,24 @@ class CLI < Thor
...
+  end
+
+  desc "clean <module>...", "Clean up compilation results for module(s)"
+  def clean(*module_names)
+    clean_stubs
+
+    FileUtils.rm_rf(ERROR_FILE)
+
+    modules = modules_from_names(module_names)
+    with_modules modules, "Cleaning up", true do |mod|
```
Comment (mvidner) [inline yk:295]: what does the boolean mean?

Comment (mvidner) [inline yk:295]: I see, `print_status`. I'd declare it as `verbosity` with a default value `:normal` and replace `false` with `:quiet`

Comment (jreidinger) [inline yk:295]: @dmajda - it is up to you, I don't create this stuff.

Comment (dmajda) [inline yk:295]: @mvidner I sacrificed a scapegoat before commiting this sin of a non-descriptive boolean parameter, but apparently it didn't help ;-)

It's not exactly a verbosity, it controls whether the final status of the action is printed. That makes sense if the action code doesn't print any other output. This is true for all actions except `compile`, which emits status messages for all files it compiles with their own statuses.

I'll fix that at some point after @jreidinger is done with all his changes related to restructuring.

**Pull request merged**